docs/features.md

```diff
@@ -9,12 +9,53 @@
 ### As a Node Operator, I do not want to expose my private wallet key on the command line
 ## Peer to Peer Network
 ### As a Node operator, I want to be able to bootstrap to the network by connecting to any known node
+#### connecting to existing node
+##### test: test/test_p2p.py::test_connecting_to_existing_node
+##### steps:
+
+* given that `standalone` is a running node in a standalone mode
+* start new node with `--bootstrap` pointing to `standalone`
+* node should succesfully start and connect to `standalone` via protocol handshake
+
+#### connecting to non-existing node
+##### test: test/test_p2p.py::test_connecting_to_non_existing_node
+##### steps:
+
+* start new node with `--bootstrap` pointing to some non-existing address
+* node should exit
+* exit code should be 1
+* node should log that bootstrap could not been found
```

Comment:
I think we shouldn't be putting corner cases like that in the feature.md file, not to mention testing them using integration tests.

Reply:
why not @ArturGajowy , wouldn't it be good to know how node behaves with weird behaviour?

Reply:
I'd say, not for the price of an integration test, most of the time. And this particular scenario I'd rather accept as 'being obvious' than wait 30s or more every build for it to be tested - even if we don't want such talk in general. If at all, this should be unit-tested.

Reply:
obvious as it may seem, currently node does not behave like describe
i don't mind for test running 3h as long as they cover functionality